### ISSUE TITLE
Word Limit setting on a field returns "NaN words left" on the front end

### ISSUE THREAD

**Issue by bloqhead** [verbb/formie#1120]
### Describe the bug

When using the "Limit" setting on a field, if I am trying to show the number of words in a field, it will output `NaN words left
`. When using the letter count instead, it functions as expected and will output `51 characters left`.

I find that if you have the words limit setting enabled on a basic textarea with no content in it, the count will show correctly (e.g. "123 words left"), but as soon as you start typing into the field, the count changes to `NaN`. For fields with rich text enabled, the word count will always show `NaN` even when you start typing into the field.

### Steps to reproduce

1. Create a form
2. Add a textarea field
3. Set a word limit under the "Settings" tab for that field
4. Try typing into the field on the front end (the count will initially appear correct until you start typing into the field)

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes ("Validate Form on Submit" and "Validate When Typing" are both enabled)
- Custom Form Templates: No

### Craft CMS version

3.7.54

### Plugin version

1.6.13

### Multi-site?

No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by bloqhead** [comment]
Awesome, thanks @engram-design!

**Comment by engram-design** [comment]
Fixed in [1.6.15](https://github.com/verbb/formie/releases/tag/1.6.15)
